examples/composite_queries/test/dfx_generated/canister3/index.js

```diff
@@ -0,0 +1,35 @@
+import { Actor, HttpAgent } from '@dfinity/agent';
```

Comment:
Please automatically generate the dfx_generated files, and do not check in dfx_generated